packages/trader/src/Modules/SmartChart/Components/AllMarkers/index.js

```diff
@@ -0,0 +1,14 @@
+// Things to do before touching this module :P
+// 1- Please read https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial
+// 2- Please read RawMarker.jsx in https://github.com/binary-com/SmartCharts
+// 3- Please read contract-store.js & trade.jsx carefully
+
+import TickContract from './tick-contract.jsx';
+import NonTickContract from './non-tick-contract.jsx';
+import DigitContract from './digit-contract.jsx';
+
+export default {
+    TickContract,
+    NonTickContract,
+    DigitContract,
+};
```

Comment:
nit: you can also re-export these imports by `export { default as TickContract } from './tick-contract.jsx';` :)